inferentia/README.md

```diff
@@ -202,4 +227,22 @@ Note:
 1. The `config.pbtxt` and `model.py` should be treated as
 starting point. The users can customize these files as per
 their need.
-2. Triton Inferentia is currently tested with a **single** model. 
\ No newline at end of file
+2. Triton Inferentia is currently tested with a **single** model. 
+
+## Testing Inferentia Setup for Accuracy
+The [qa folder](https://github.com/triton-inference-server/python_backend/tree/main/inferentia/qa)
+contains the necessary files to set up testing with a simple add_sub model. The test
+requires an instance with more than 8 inferentia cores to run, eg:`inf1.6xlarge`.
+start the test, run 
+```
+source <triton path>/python_backend/inferentia/qa/setup_test_enviroment_and_test.sh

```

Comment:
Let's use a consistent style. Following the ReadMe in the repo https://github.com/triton-inference-server/python_backend#quick-start

All the shell command must be preceded by $.